### DISCUSSION TITLE
Could not find production build in the /app/.next directory

### DISCUSSION THREAD

**Discussion by katiesun138** [vercel/next.js#54390] [Help]
### Summary

Hi everyone,
I am currently trying to run my application using Docker. So far I am stuck with the error message:
```
Error: Could not find a production build in the '/app/.next' directory. Try building your app with 'next build' before starting the production server. https://nextjs.org/docs/messages/production-start-no-build-id
```
I have tried deleting my .next folder and running ```npm run build```. After, when I run ```docker-compose up```, it still results in the same issue. I am using next: 13.4.6.

Here is my Dockerfile:
```

FROM node:alpine

WORKDIR /app

COPY package*.json ./
COPY prisma ./prisma/
COPY .env ./
COPY tsconfig.json ./

RUN npm install
RUN npx prisma generate

COPY next.config.js ./next.config.js
COPY . .

EXPOSE 3001

CMD npm start
```

Here are my scripts within the package.json file:
```
...
"scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "generate": "npx prisma generate",
    "deploy": "npx prisma migrate deploy"
  },
...
```

Any suggestions would be helpful. Thank you in advance!

**Comment by bbergeron-xlkey** [comment]
Hi, I had a similar issue and found a solution: https://github.com/vercel/next.js/discussions/69740
